client/state/selectors/get-site-comments.js

```diff
@@ -20,16 +20,17 @@ function filterCommentsByStatus( comments, status ) {
  * @param {Object} state Redux state
  * @param {Number} siteId Site for whose comments to find
  * @param {String} [status=unapproved] Status to filter comments
+ * @param {String} [order=asc] Order in which to sort filtered comments
  * @returns {Array<Object>} Available comments for site, filtered by status
  */
 export const getSiteComments = createSelector(
-	( state, siteId, status = 'unapproved' ) => {
+	( state, siteId, status = 'unapproved', order = 'asc' ) => {
 		const comments = get( state, 'comments.items', {} );
 		const parsedComments = Object.keys( comments )
 			.filter( key => parseInt( key.split( '-', 1 ), 10 ) === siteId )
 			.reduce( ( list, key ) => [ ...list, ...comments[ key ] ], [] );
 
-		return filterCommentsByStatus( parsedComments, status );
+		return orderBy( filterCommentsByStatus( parsedComments, status ), 'date', order );
```

Comment:
this selector is starting to get heavy and we should start thinking about memoizing it. that being said I don't think this is the time to do that. we will notice it later when we have thousands of comments pouring in.